digits/model/tasks/caffe_train.py

```diff
@@ -39,6 +39,50 @@
 CAFFE_DEPLOY_FILE = 'deploy.prototxt'
 
 @subclass
+class DigitsTransformer(caffe.io.Transformer):
```

Comment:
> Probably the worst that would happen is we wouldn't make use of some new functionality or bugfix in preprocess().

I am calling `caffe.io.Transformer.preprocess()` on line 82 - wouldn't we then benefit from a bug fix in this function?

If `resize==True` (the default) then I believe we are doing exactly the same thing as before. If `resize==False` then in this class's `preprocess()` the dimensions of `self.inputs[in_]` are changed to avoid resizing the input data and if we are doing mean image subtraction then the mean image is resized to the dimensions of the input data.